webapp/src/components/data/dataset_input/DatasetInput.vue

```diff
@@ -202,9 +207,9 @@ interface Props {
   datasetBuilder: data.DatasetBuilder<File>
   isOnlyPrediction: boolean //is true when predicting with a trained model
 }
-
 const props = defineProps<Props>()
 
+const browsingTip = "Tip: when browsing files you can use the keyboard shortcut Ctrl + A to select all images."
```

Comment:
let's inline it (only used once, is not changing)

Reply:
Good catch! it was supposed to be used twice! It got lost in some change reversion.

You were saying your limit is 3, mine is at 2 in this case. I believe that someone other than me would not necessarily notice it needs to be changed in 2 different places, especially if it's a quick change to reword or fix a typo.